### PR TITLE
FS-2879: Make urls dynamic in create_app

### PR DESCRIPTION
**Pull request by RamuniN**
https://digital.dclg.gov.uk/jira/browse/FS-2879

### Description
Make contact_us page & service names in banner dynamic in the tasklist page

### Screenshots of UI changes (if applicable)
image
image

### CODE REVIEW

**Review by tferns — APPROVED** [review]
Small minor suggestion but LGTM 👍

File: app/create_app.py [hunk 1]
```diff
@@ -181,26 +198,39 @@ def inject_service_name():
...
+            if not fund:
+                application_id = (
+                    request.view_args.get("application_id")
+                    if request.view_args.get("application_id")
+                    else request.form["application_id"]
+                )
+                application = get_application_data(
+                    application_id, as_dict=True
+                )
+                fund = get_fund_data(
```
Comment (srh-sloan) [inline app/create_app.py:210]: Should probably check if application is not None here

File: app/create_app.py [hunk 2]
```diff
@@ -160,9 +162,24 @@ def find_round_in_request(fund):
...
-        if request.view_args or request.args:
+        if request.view_args or request.args or request.form:
             try:
                 fund = find_fund_in_request()
+                if not fund:
+                    application_id = (
+                        request.view_args.get("application_id")
+                        if request.view_args.get("application_id")
+                        else request.form["application_id"]
+                    )
```
[the remark this answers is not in the document]
Comment (RamuniN) [reply app/create_app.py]: Done @tferns

File: app/create_app.py (hunk 1 above)
Comment (RamuniN) [reply app/create_app.py:208]: Here I want to catch exception if application is None. so don't want to check

**Review by tferns — APPROVED** [review]

**Pull request merged**